cardinal/plugin_receipt.go

```diff
@@ -56,14 +56,14 @@ func (p *receiptPlugin) RegisterQueries(world *World) error {
 
 // queryReceipts godoc
 //
-//	@Summary		Get transaction receipts from Cardinal
-//	@Description	Get transaction receipts from Cardinal
-//	@Accept			application/json
-//	@Produce		application/json
-//	@Param			ListTxReceiptsRequest	body		ListTxReceiptsRequest	true	"List Transaction Receipts Request"
-//	@Success		200						{object}	ListTxReceiptsResponse
-//	@Failure		400						{string}	string	"Invalid transaction request"
-//	@Router			/query/receipts/list [post]
+//	@Summary      Get transaction receipts from Cardinal
+//	@Description  Retrieves all transaction receipts from Cardinal
```

Comment:
The summary and description say the same thing: I'd much rather it be uniform. The description you wrote should be used for summary here.